### PR TITLE
Wallpapers overhaul

### PR DESCRIPTION
**Pull request by win8linux**
This MR is in preparation for a community wallpapers package to be put on HaikuPorts.

# Change summary

## Additions
* IQ: GIMP project file
* 5K resolution wallpapers of Andrii Maykovsky wallpapers (Difference, Consistency, Simplicity, Accuracy, Honesty, Artistry)
* Autumn Leaves: 5K resolution wallpaper (with fixed Haiku logo)
* Various copyright updates where needed
* Pre-existing Krita and GIMP project files updated to 5K resolution
* Animated WebP previews of wallpapers with multiple variations
* Simpliteaf: Add wallpaper (originally Cartoon Network Leaks)

### Simple White
* Missing HDRI file needed by Simple White's Blender project added
* Rerendered 5K resolution wallpaper

### transformat
* Teapot and A Glitch: GIMP project file
* Teapot and A Glitch: 5K resolution wallpaper

### Braelin Michelus wallpapers
* Beauty in Simplicity: 5K resolution wallpapers
* Beauty in Simplicity: GIMP project files
* Shining Harmony: 5K resolution wallpaper
* Shining Harmony: GIMP project file
* Yesterday's Future: 5K resolution wallpaper
* Yesterdays Future: GIMP project file
* Copyright

## Haiku Blue
* 5K resolution wallpaper
* Copyright

## Removals
* Cartoon Network Leaks (possible legal issue with name)
> `Cartoon Network` is a property of The Cartoon Network, Inc. (part of Warner Bros. Discovery)
* Haiku Fragmented
> Source SVG cannot be properly displayed by any program running in Haiku
* 3K resolution wallpapers from Andrii Maykovsky removed

### CODE REVIEW

**Comment by humdingerb** [comment]
LGTM 👍

**Comment by win8linux** [comment]
Cartoon Network Leaks just got back to me, giving permission to rename their wallpaper to something else. Any suggestions?

**Comment by humdingerb** [comment]
Simpliteaf.

**Comment by kallisti5** [comment]
Can this one be squashed?  There's a lot of commits here for a few wallpaper with additions and removals.  It's hard to follow.

**Comment by win8linux** [comment]
I'll attempt to squash them, but a reason why the commits were split was to ease pushing them individually to remote. A prior branch couldn't be pushed due to the connection with remote dropping while pushing the larger commits.

**Comment by kallisti5** [comment]
Thanks, much easier to understand now :-)

Overall looks good.   Are the copyrights / licenses clear on all of these?  Not seeing wording with all of them.

**Comment by win8linux** [comment]
They should all be either CC-BY or CC-BY-SA 4.0, as mentioned in the INFO files. The previously (possibly) legally problematic submission (Cartoon Network Leaks) was changed with permission from the submitter.

**Comment by kallisti5** [comment]
👍  sounds good.  Thanks!
